packages/app/src/specs/generators/empty/EmptyGeneratorCard.cy.ts

```diff
@@ -0,0 +1,9 @@
+import { mount } from '@cypress/vue'
+import Emptygeneratorcard from './EmptyGeneratorCard.vue'
+
+describe('<Emptygeneratorcard />', () => {
+  it('renders', () => {
+    // see: https://vue-test-utils.vuejs.org/
+    mount(Emptygeneratorcard, { props: {} })
```

Comment:
Might be good to use the correct casing here, EmptyGeneratorCard

Reply:
this should not exist. accidentally committed while testing the generator.